frontend/src/global_styles/openproject/_mixins.sass

```diff
@@ -222,8 +222,9 @@ $scrollbar-size: 10px
     padding-bottom: 0
 
 @mixin extended-content--top
-  #content-body
-    padding-top: 0
+  @media screen and (min-width: $breakpoint-sm)
```

Comment:
I'm afraid of global changes. How can I know that this is not going to have collateral effects somewhere else?